.github/workflows/ci.yml

```diff
@@ -14,40 +14,28 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  dedupe:
+  lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - name: Use Node.js 18.x
         uses: actions/setup-node@v3
         with:
           node-version: '18.x'
+          cache: 'yarn'
+      - uses: actions/cache@v3
+        id: cache
+        with:
+          path: |
+            node_modules
+            */**/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
       - name: Run yarn dedupe
         run: yarn dedupe --check
-
-  format:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js 18.x
-        uses: actions/setup-node@v3
-        with:
-          node-version: '18.x'
       - name: Install dependencies
-        run: yarn install --immutable --immutable-cache
```

Comment:
We want CI jobs to fail if a change to the cache would be made as a result from this install. This helps ensure integrity of dependencies between local dev environments and CI.

Reply:
still does. yarn knows if it's running in a CI